tests/features/common/kie-common.feature

```diff
@@ -23,6 +23,7 @@ Feature: RHPAM and RHDM common tests
       | AUTH_LDAP_DEFAULT_ROLE                        | test                         |
       | AUTH_LDAP_DISTINGUISHED_NAME_ATTRIBUTE        | name1                        |
       | AUTH_LDAP_JAAS_SECURITY_DOMAIN                | other                        |
+      | AUTH_LDAP_LOGIN_MODULE                        | required                     |
```

Comment:
It defaults to required. so 2 testes needed, passing the optional flag and not setting it and check for the default value.

where is the verification of this new env?
and file .... should contain ...  <login-module code="LdapExtended" flag="optional">